nova/views.py

```diff
@@ -560,13 +560,10 @@ def create_group():
     return render_template('group/create.html', form=form)
 
 
-@app.route('/wave')
+@app.route('/dataset/<user>/<dataset>/wave')
 @login_required(admin=False)
-def wave_it():
+def wave(user, dataset, path=''):
```

Comment:
Which means you can also remove the `path` argument ;-)

Reply:
oops :sweat_smile:  done!